opm/autodiff/BlackoilDetails.hpp

```diff
@@ -195,7 +195,60 @@ namespace detail {
             std::size_t count = countLocalInteriorCells(grid);
             return grid.comm().sum(count);
         }
-
+        /// \brief Find the rows corresponding to overlap cells
+        ///
+        /// Loop over grid and store cell ids of row-column pairs 
+        /// corrsponding to overlap cells.
+        /// \tparam The type of the DUNE grid.
+        /// \param grid The grid where we look for overlap cells.
+        /// \param overlapRowAndColumns List where overlap rows and columns are stored.
+        template<class Grid>
+	void findOverlapRowsAndColumns(const Grid & grid,std::vector<std::pair<int,std::vector<int>>> &overlapRowAndColumns )
+	{
+	  //only relevant in parallel case.
+	  if ( grid.comm().size() > 1)
+	    {
```

Comment:
Either put the brace on the same line as the `if`, or indent it to the same level as the `if`. Use 4 spaces for indentation per level.